guides/common/attributes-base.adoc

```diff
@@ -138,6 +138,8 @@
 :RHEL: Red{nbsp}Hat Enterprise Linux
 :RHELServer: {RHEL} Server
 :SLES: SUSE Linux Enterprise Server
+:salt-documentation-title: official Salt documentation
```

Comment:
Is this necessary? Any attribute introduces additional complexity, which should be outweighed by the benefits said attribute is expected to add. In this case, I'd argue that "official <project> documentation" is just a regular group of words and doesn't justify an attribute.

Reply:
Removed.